pom.xml

```diff
@@ -6,7 +6,7 @@
   <packaging>jar</packaging>
   <version>1.0-SNAPSHOT</version>
   <name>my-app</name>
-  <url>http://maven.apache.org</url>
+  <url>https://github.com/microsoft/vscode-remote-try-java</url>
```

Comment:
What does this URL affect? Why should we avoid pointing to maven.apache.org?

Reply:
This should point to the project's URL, so in that case https://github.com/microsoft/vscode-remote-try-java
Those are metadata for the project, like its name and description

Reply:
Makes sense, thanks for explaining!